cpp_linter/rest_api/github_api.py

```diff
@@ -502,7 +570,7 @@ def create_review_comments(
                 file_obj, summary_only, review_comments
             )
 
-    def _dismiss_stale_reviews(self, url: str):
+    def _dismiss_stale_reviews(self, url: str, ignored_reviews: List[str] = None):
         """Dismiss all reviews that were previously created by cpp-linter"""
```

Comment:
`None` is not a type of list.
```suggestion
    def _dismiss_stale_reviews(self, url: str, ignored_reviews: Optional[List[str]] = None):
        """Dismiss reviews that were previously created by cpp-linter.
        
        Use ``ignored_reviews`` to only dismiss the reviews not specified in
        the given list. If ``ignored_reviews`` is `None`, then dismiss all
        reviews.
        """
```
I also added some info about the new parameter in the doc string.